CHANGELOG.rst

```diff
@@ -1,5 +1,52 @@
 Changelog
 =========
+1.8.0 (2020-04-16)
+------------------
+MLflow 1.8.0 includes several major features and improvements:
+
+Features:
+
+- Added new APIs for Azure Deployment (#2375 @csteegz, #2711, @akshaya-a)
+- Added support for case sensitive LIKE and ILIKE operator in SQLite (#2708, @mparkhe)
+- Added line smoothing on Exponential Moving Average (#2620, @Valentyn1997)
+- Adding support for spacy flavor (#2242, @arocketman)
+- Separating selection checkbox in mlflow UI to have it's own column (#2559, @harupy)
```

Comment:
I would actually move this one to bug fixes, and say it "Fixes a bug where hiding the start time column in the UI would also hide run selection checkboxes"

Reply:
Done